components/ComponentViewer/index.jsx

```diff
@@ -0,0 +1,9 @@
+const ComponentViewer = ({ children, componentName, componentDescription }) => (
```

Comment:
```suggestion
const ComponentViewer = ({ children, name, description }) => (
```
"componentName" and "componentDescription" I think are a bit redundant considering the name of the component. What do you think about removing the "component" prefix?